include/pg_tmgr.h

```diff
@@ -0,0 +1,28 @@
+/*-------------------------------------------------------------------------
+ *
+ * include/pg_tmgr.h
+ *
+ * Transaction manager API
+ *
+ * Copyright (c) 2014-2015, Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#ifndef PG_TMGR_H
+#define PG_TMGR_H
+
+typedef struct { 
+	bool (*Begin)(PGconn* conn);
+	bool (*Prepare)(PGconn* conn, char const* relationName, ShardId shardId);
```

Comment:
Frankly speaking "porefix" means nothing (especially in this context) and transactionId is actually not correct, because transationId is formed from relation name and shardId.
So my interpretation was that this function is given relation-name and shard-id and can perform with them anything it like, for example form transaction-id if it is needed. It is responsibility of this function to construct unique transaction id rather than of caller of this function.